spec/system/admin/products_v3/products_spec.rb

```diff
@@ -354,231 +324,306 @@
     end
   end
 
-  describe "Cloning Feature" do
-    let!(:variant_a1) {
-      create(:variant,
-             product: product_a,
-             display_name: "Medium box",
-             sku: "APL-01",
-             price: 5.25)
-    }
-    let(:product_a) { create(:simple_product, name: "Apples", sku: "APL-00") }
-
-    before do
-      visit admin_products_url
-    end
+  describe "edit image" do
+    shared_examples "updating image" do
+      it "saves product image" do
+        visit admin_products_url
 
-    describe "Actions columns (clone)" do
-      it "shows an actions menu with a clone link when clicking on icon for product" do
         within row_containing_name("Apples") do
-          page.find(".vertical-ellipsis-menu").click
-          expect(page).to have_link "Clone", href: spree.clone_admin_product_path(product_a)
-        end
-
-        within row_containing_name("Medium box") do
-          page.find(".vertical-ellipsis-menu").click
-          expect(page).to_not have_link "Clone", href: spree.clone_admin_product_path(product_a)
+          click_on "Edit"
         end
-      end
-    end
-
-    describe "Cloning product" do
-      it "shows the cloned product on page when clicked on the cloned option" do
-        within "table.products" do
-          # Gather input values, because page.content doesn't include them.
-          input_content = page.find_all('input[type=text]').map(&:value).join
 
-          # Products does not include the cloned product.
-          expect(input_content).to_not match /COPY OF Apples/
-        end
+        within ".reveal-modal" do
+          expect(page).to have_content "Edit product photo"
+          expect_page_to_have_image(current_img_url)
 
-        within row_containing_name("Apples") do
-          page.find(".vertical-ellipsis-menu").click
-          click_link('Clone')
+          # Upload a new image file
+          attach_file 'image[attachment]', Rails.public_path.join('500.jpg'), visible: false
```

Comment:
I think that it would be better style use a standard spec helper:
```suggestion
          attach_file 'image[attachment]', file_fixture("logo.png"), visible: false
```